Spanner/src/Session/CacheSessionPool.php

```diff
@@ -650,18 +652,36 @@ private function getSession(array &$data)
      */
     private function createSessions($count)
     {
-        $sessions = [];
+        $options = [];
+        if ($this->config['labels']) {
+            $options['labels'] = $this->config['labels'];
+        }
+
+        $args = [
+                'database' => $this->database->name(),
+                'session' => [
+                    'labels' => $this->pluck('labels', $options, false) ?: []
+                ]
+            ] + $options;
```

Comment:
`$options` will always be empty once you've plucked labels out of it, no?

Why not replace line 663 with the following, and don't bother with `$options` unless it'll be used for other things?

```php
'labels' => isset($this->config['labels']) ? $this->config['labels'] : []
```

Added benefit of not needing to import the trait.

Reply:
Sounds great to me.